### ISSUE TITLE
Add configuration properties for file download

### ISSUE THREAD

**Issue by bechte** [sap-commerce-tools/ccv2-project-template#18]
In 
https://github.com/sap-commerce-tools/ccv2-project-template/blob/10ea046ce2469773d7d56ab41df256c118c7d8ca/core-customize/build.gradle.kts#L28

The download URL of the SAP Software Download is hard-coded for the underlying SAP commerce version. This parameter needs to be modified and updated with the version number in the manifest.json. This potentially leads to issues / crashed due to forgotten updates.

I wonder if we could replace the hard coded value with a configuration value that we place into a gradle.properties file.

```
com.sap.cx.commerce.downloads.2011.9.file=https://softwaredownloads.sap.com/file/0020000000989902021
com.sap.cx.commerce.downloads.2105.0.file=https://softwaredownloads.sap.com/file/0020000001077242021
```

The same could / should be done for the integration packs. Maybe it's even better to place the mapping in an external file, but this could also be a step 2.

**Comment by mpern** [comment]
that's a good idea.
fail as soon as the version in the manifest doesn't match the download

**Comment by bechte** [comment]
solved with pull-request #22
